src/fragments/lib-v1/storage/js/getting-started.mdx

```diff
@@ -0,0 +1,264 @@
+      ],
+      "Effect": "Allow"
+    },
+    {
+      "Condition": {
+        "StringLike": {
+          "s3:prefix": [
+            "public/",
+            "public/*",
+            "protected/",
+            "protected/*",
+            "private/${cognito-identity.amazonaws.com:sub}/",
+            "private/${cognito-identity.amazonaws.com:sub}/*"
+          ]
+        }
+      },
+      "Action": [
+        "s3:ListBucket"
+      ],
+      "Resource": [
+        "arn:aws:s3:::{enter bucket name}"
+      ],
+      "Effect": "Allow"
+    }
+  ]
+}
+```
+
+Inline policy for the `Unauth_Role`:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "s3:GetObject",
+        "s3:PutObject",
+        "s3:DeleteObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::{enter bucket name}/public/*"
+      ],
+      "Effect": "Allow"
+    },
+    {
+      "Action": [
+        "s3:PutObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::{enter bucket name}/uploads/*"
+      ],
+      "Effect": "Allow"
+    },
+    {
+      "Action": [
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::{enter bucket name}/protected/*"
+      ],
+      "Effect": "Allow"
+    },
+    {
+      "Condition": {
+        "StringLike": {
+          "s3:prefix": [
+            "public/",
+            "public/*",
+            "protected/",
+            "protected/*"
+          ]
+        }
+      },
+      "Action": [
+        "s3:ListBucket"
+      ],
+      "Resource": [
+        "arn:aws:s3:::{enter bucket name}"
+      ],
+      "Effect": "Allow"
+    }
+  ]
+}
+```
+
+The policy template that Amplify CLI uses is found [here](https://github.com/aws-amplify/amplify-cli/blob/b12d20b9d85f7fc6abf7e2f7fbe11e1a108911b9/packages/amplify-category-storage/provider-utils/awscloudformation/cloudformation-templates/s3-cloudformation-template.json).
+
+### Amazon S3 Bucket CORS Policy Setup
+<!-- TODO Rewrite is this needed only for JS? or is it needed for Flutter Web for example as well? -->
+<Callout warning>
+
+To make calls to your S3 bucket from your App, you need to set up a CORS Policy for your S3 bucket. This callout is only for manual configuration of your S3 bucket, CORS Policy configuration is done automatically via Amplify CLI when running `amplify add storage`.
+
+</Callout>
+
+The following steps will set up your CORS Policy: 
+
+1. Go to [Amazon S3 Console](https://s3.console.aws.amazon.com/s3/home?region=us-east-1) and click on your project's `userfiles` bucket, which is normally named as [Bucket Name][Id]-dev. 
+![Go to [Amazon S3 Console]](/images/storage/CORS1.png)
+2. Click on the **Permissions** tab for your bucket, and then click on the **Cross-origin resource sharing (CORS)** tile.
+![Click on the **Permissions** tab for your bucket](/images/storage/CORS2.png)
+3. Update your bucket's CORS Policy to look like:
+![Update your bucket's CORS Policy to look like:](/images/storage/CORS3.png)
```

Comment:
Since this alt text is the same as the text above and you already show the code below the picture, you could probably make it be a decorative image like this:

```suggestion
![](/images/storage/CORS3.png)
```

This way when using screen readers, it's not reading out the sentence `Update your bucket's CORS Policy to look like:` twice

Reply:
Removed in latest diff